src/core/css/inc/mixins.scss

```diff
@@ -67,3 +69,129 @@
       + 'Please make sure it is defined in `$breakpoints` map.';
   }
 }
+
+/* Arrows */
+$default-arrow-margin: 3px;
+@mixin arrow($margin: $default-arrow-margin) {
+  background: url('~amo/img/icons/arrow.svg') 50% 50% no-repeat;
```

Comment:
Yeah I would like to improve our icon/SVG game, but have kept it as an idea for a lazy Friday/Sunday.